batch2/create-batch-tables.sql

```diff
@@ -33,6 +33,11 @@ CREATE TABLE IF NOT EXISTS `batches` (
   `n_failed` INT NOT NULL DEFAULT 0,
   `n_cancelled` INT NOT NULL DEFAULT 0,
   `time_created` TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+  `time_completed` TIMESTAMP,
+  `cost` DOUBLE NOT NULL DEFAULT 0,
+  `running_cost` DOUBLE NOT NULL DEFAULT 0,
```

Comment:
Can you comment of these are for future use?

Or just delete them.  It isn't necessary.  I don't mind blowing away the database for now, and we'll need to stop doing it completely soon anyway.  I'm working on a database migration plan for that.